docs/presentation/how-to-add-a-video-to-a-slide-in-a-presentation.md

```diff
@@ -0,0 +1,138 @@
+---
+
+api_name:
+- Microsoft.Office.DocumentFormat.OpenXML.Packaging
+api_type:
+- schema
+ms.assetid: 403abe97-7ab2-40ba-92c0-d6312a6d10c8
+title: 'How to: Add a video to a slide in a presentation'
+ms.suite: office
+
+ms.author: o365devx
+author: o365devx
+ms.topic: conceptual
+ms.date: 03/31/2025
+ms.localizationpriority: medium
+---
+
+# Add a video to a slide in a presentation
+
+This topic shows how to use the classes in the Open XML SDK for
+Office to add a video to the first slide in a presentation
+programmatically.
+
+## Getting a Presentation Object 
+
+In the Open XML SDK, the `PresentationDocument` class represents a
```

Comment:
I think there's a shared snippet that could be used here that has links to the APIs. @mikeebowen do you know what I'm referring to?

Reply:
Nothing that is a match for this, but if this gets repeated, we should create one